build.gradle

```diff
@@ -67,7 +67,8 @@ artifactory {
 }
 
 dependencies {
-    compile 'com.blackducksoftware.integration:hub-common:26.1.1'
+    compile 'com.blackducksoftware.integration:hub-common:27.0.0-SNAPSHOT'
+    compile 'org.apache.httpcomponents:httpclient:4.5.5'
```

Comment:
Detect is resolving to httpclient  4.5.3 and we need 4.5.5, we should investigate whats going on. It was not immediately obvious to me